content/docs/developing-operators.md

```diff
@@ -136,22 +76,11 @@ Plan foo
 
 Plans consists of one or more `phases`. `Phases` consists of one or more `steps`. `Steps` contain one or more `tasks` (those are defined in the section we talked about in the last paragraph). Both phases and also steps can be configured with an execution `strategy`, either `serial` or `parallel`.
 
-The sample has a `deploy` plan with a `deploy-phase` and a `deploy-step`. From the `deploy-step` the `deploy-task` is referenced. This task gets executed when an instance is created using the operator.
+The sample has a `deploy` plan with a `main` and a `everything`. From `everything` the `app` is referenced. This task gets executed when an instance is created using the operator.
```

Comment:
```suggestion
The sample has a `deploy` plan with a phase `main` and a step `everything`. From `everything` the `app` is referenced. This task gets executed when an instance is created using the operator.
```

Reply:
One downside of including files is that someone might change IDs in the file which leads to weird documentation. I don't think we can prevent this though and the benefit of including them weighs heavier I think.